### PR TITLE
No export option

### PR DESCRIPTION
**Pull request by tkarna**
Test cases should not create any files.

### CODE REVIEW

File: test/bottomFriction/test_implicit_friction_parabolic.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def test_implicit_friction_parabolic(do_assert=True):
     options.t_export = t_export
     options.dt = dt
     options.t_end = t_end
+    options.no_exports = ~do_export
```
Comment (wence-) [inline test/bottomFriction/test_implicit_friction_parabolic.py]: Idiomatically one would write `not do_exports`

**Comment by wence-** [comment]
Looks sane

File: test/bottomFriction/test_implicit_friction_parabolic.py (hunk 1 above)
Comment (tkarna) [inline test/bottomFriction/test_implicit_friction_parabolic.py]: Yes you are right.

**Comment by tkarna** [comment]
Verified that py.test does not create any files on my laptop, should be good to go.

**Pull request merged**